tools/acl_compressor/sources/main.cpp

```diff
@@ -247,7 +280,23 @@ static void try_algorithm(const Options& options, Allocator& allocator, const An
 
 		ACL_ENSURE(compressed_clip->is_valid(true), "Compressed clip is invalid");
 
-		unit_test(allocator, clip, skeleton, *compressed_clip, algorithm);
+		if (DECOMPRESSION_TIME_EVALUATIONS == 0)
```

Comment:
as discussed, ideally we want to time the decompression on multiple sample times since it might be faster or slower to sample early in a segment/clip or late, idem with forward/backwards/random access